readthedocs/proxito/views/hosting.py

```diff
@@ -256,16 +272,16 @@ class BuildSerializerNoLinks(NoLinksMixin, BuildSerializer):
     pass
 
 
-class AddonsResponse:
+class AddonsResponseBase:
     def get(
         self,
         addons_version,
         project,
+        request,
```

Comment:
There is no need to receive the `request` here and passing it to the other methods. We just need the `user`; which is the only attribute we are using from the `request`.

Reply:
A request object is needed for .com,see https://github.com/readthedocs/readthedocs-corporate/pull/1857.